authors/apps/authentication/views.py

```diff
@@ -17,7 +20,7 @@ class RegistrationAPIView(APIView):
     serializer_class = RegistrationSerializer
 
     def post(self, request):
-        user = request.data.get('user', {})
+        user = request.data
```

Comment:
@BarnaTB This changes the type of json object sent, was this agreed upon by the team.

Reply:
No. We've not had the discussion yet. I will raise it up to the rest of the team on Monday.